src/maintain/disabled.js

```diff
@@ -20,11 +20,19 @@ import queryFocusable from '../query/focusable';
 import elementDisabled from '../element/disabled';
 import {getParentComparator} from '../util/compare-position';
 
+let inertElementSet = [];
```

Comment:
you can have multiple instances of InertSubtree and the list of elements belongs on that class.